cmd/onomy-load-test/README.md

```diff
@@ -0,0 +1,131 @@
+# tm-load-test custom client for cosmos chain.
```

Comment:
!! @puneetsingh166  Sorry for the late comment, but better move this "customClient" content to onomy tm-load-test, and add this repo as a dependency in gomod. as a result you will not have to copy constants on onomy and will be possible to update the test in onomy repo.  The comments abow still actual anyway !!!